examples/redis/chart.test.ts

```diff
@@ -0,0 +1,29 @@
+import { RedisChart } from "./chart";
+import { Testing } from "cdk8s";
+import { TalisShortRegion, TalisDeploymentEnvironment } from "../../lib";
+
+describe("Redis example", () => {
+  const PROCESS_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...PROCESS_ENV };
+    process.env.DOCKER_USERNAME = "someuser";
+    process.env.DOCKER_PASSWORD = "secret123";
```

Comment:
The construct uses a public image and doesn't use imagePullSecrets, no need for these variables.